src/styles/1-helpers/_common.scss

```diff
@@ -0,0 +1,16 @@
+html {
+  font-size: $fz;
+  font-family: "Roboto", "Helvetica", sans-serif;
```

Comment:
Now everything is done correctly, but it is a good practice to also put the `font-family` in a separate variable.